blocks/header/header.css

```diff
@@ -1,13 +1,12 @@
 /* header and nav layout */
```

Comment:
1. Looking at the [Frontify specs](https://sap.frontify.com/document/223143#/-/masthead), it seems a blue underline needs to come for the selected item.

![image](https://github.com/urfuwo/hlx-test/assets/1191451/41921d4b-2427-4130-8821-daaa77d47736)

2. Also the background color should be light grey.
3. The top/bottom paddings looks a little bit less than specs.
4. The left/right margins of the header look different from the page. Probably they should align though I didn't find that spelled out in the specs.

Reply:
@sdmcraft Thanks for noticing the colour. I changed the background and pushed changes. 
For active section marker we need the site structure, so navigation know which page is active then make the active section underline blue. Hence not been implemented now.

Reply:
Paddings and margins are referred from figma:
https://www.figma.com/file/OFEfvhyXJvYJ6MQZ30699m/v2.0-Brand-Digital-Design-System?type=design&node-id=8969%3A77791&mode=dev

Reply:
ok thanks. On point 4 above I think we need to align header with rest of the page but can be taken up separately.